test-files/cmd/glacial/NonDestructiveConflationTest.sh.stdout

```diff
@@ -1,155 +1,74 @@
14:46:37.281 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 40 / 117        
14:46:37.281 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 41 / 117        
14:46:37.281 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 42 / 117        
14:46:37.281 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 43 / 117        
14:46:37.283 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 44 / 117        
14:46:37.284 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 45 / 117        
14:46:37.284 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 46 / 117        
14:46:37.284 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 47 / 117        
14:46:37.285 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 48 / 117        
14:46:37.286 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 49 / 117        
14:46:37.287 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 50 / 117        
14:46:37.287 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 51 / 117        
14:46:37.287 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 52 / 117        
14:46:37.288 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 53 / 117        
14:46:37.289 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 54 / 117        
14:46:37.290 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 55 / 117        
14:46:37.291 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 56 / 117        
14:46:37.292 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 57 / 117        
14:46:37.292 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 58 / 117        
14:46:37.293 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 59 / 117        
14:46:37.293 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 60 / 117        
14:46:37.294 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 61 / 117        
14:46:37.294 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 62 / 117        
14:46:37.295 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 63 / 117        
14:46:37.296 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 64 / 117        
14:46:37.297 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 65 / 117        
14:46:37.299 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 66 / 117        
14:46:37.300 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 67 / 117        
14:46:37.300 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 68 / 117        
14:46:37.301 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 69 / 117        
14:46:37.301 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 70 / 117        
14:46:37.301 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 71 / 117        
14:46:37.303 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 72 / 117        
14:46:37.303 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 73 / 117        
14:46:37.304 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 74 / 117        
14:46:37.304 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 75 / 117        
14:46:37.304 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 76 / 117        
14:46:37.304 INFO  ...rk/ConflictsNetworkMatcher.cpp( 270) 77 / 117        
14:46:37.304 INFO  ...rk/ConflictsNetworkMatcher.cpp( 2
```

Comment:
This is interesting that it created fewer nodes and ways but didn't change the number modified or deleted.